.eleventy.js

```diff
@@ -19,6 +21,9 @@ module.exports = function (config) {
   // Copy USWDS init JS so we can load it in HEAD to prevent banner flashing
   config.addPassthroughCopy({'./node_modules/@uswds/uswds/dist/js/uswds-init.js': 'assets/js/uswds-init.js'});
 
+  // Specific scripts to guides
+  config.addPassthroughCopy("./assets/_common/js/flashing.js");
+  config.addPassthroughCopy("./assets/_common/js/time-outs.js");
```

Comment:
I'm not sure adding these scripts here and in the assets/common/js is the best move... Because it exposes them to all the guides. But I'm not sure what an alternative pattern for stuff like this should be. @geekygirlsarah or @igorkorenfeld any ideas?

Reply:
To your point, I wonder if we could have individual stylesheets or script tags load for guides as they need them. I am not totally sure how we might do that (besides perhaps adding another data file).  

Just for organization, I think we should put it under the guide subfolder like `/assets/accessibility/js`.